libvast/src/system/indexer_downstream_manager.cpp

```diff
@@ -115,15 +121,15 @@ void indexer_downstream_manager::cleanup_partition(set_type::iterator& it) {
       if (f == paths_.end()) {
         VAST_WARNING(self(), "tries to delete a non-existing outbound path");
         // We could just exit now, but maybe something unforseen happened to
-        // that one INDEXER while the rest of the partiton is ok, so we rather
+        // that one INDEXER while the rest of the partiton is ok, so we better
         // try to clean up the others.
         f = paths_.begin();
         continue;
       }
     }
+    about_to_erase(f->second.get(), false, nullptr);
```

Comment:
We usually have verbs for function names. This sounds like a property, so the reader doesn't know what the function is *doing*. Here's a suggestion for what it *could* be doing based on reading the call site:

```suggestion
    schedule_removal(f->second.get(), false, nullptr);
```

Reply:
This is a function from CAF that needs to be called. Nothing I can change.